### ISSUE TITLE
Unreleased Pokemon?

### ISSUE THREAD

**Issue by Yo-Cookie** [supercarotte/PSSE#10]
Are You Able To Obtain And Use Unreleased Pokemon?

**Comment by WubTheGame** [comment]
Sorry to comment, but how do i use this? There is no .exe and i have no clue what to do.

**Comment by supercarotte** [comment]
Yes, though most unreleased pokemons are placeholders (Normal, Opportunist, 30BP) unless they are planned to be released soon (like pokemons from event stages set to appear during the current game version's lifespan, for example).
If you want a specific Pokemon, select it in the list and click its "Caught" checkbox like any other one. There are also Bulk options but these aren't in the pre-compiled releases.

**Comment by Auscobe** [comment]
I apologize for commenting here as i know and understand that this is not GBATemp, but how do you use this? there is no EXE and i have no clue what to do. if i have to compile it myself i don't know how.
